config/locales/mailers/en.yml

```diff
@@ -39,6 +39,9 @@ en:
       subject: "[%{app_name}] Your AO3 admin account"
       url: Admin login URL
       username: Admin username
+    totp_2fa_backup_codes:
+      created: A set of new backup codes has been requested for your account. Please use one of the following to log in, and then set up two-factor authentication again.
```

Comment:
For me this implies that entering a backup code will disable 2FA, but it doesn't. It just enables you to log in and then disable/re-enable it (if you want to) TODO Bilka see this convo https://github.com/otwcode/otwarchive/pull/5136#discussion_r2083320481, probably fine?

See the other TODO Bilka comment, maybe it does disable 2FA but doesn't update the state I can see from my preferences when I try to disable it there?

Reply:
Added a small clarification; what do you think?